nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/nf-registry.html

```diff
@@ -79,6 +79,9 @@
             <div id="nifi-registry-documentation" *ngIf="nfRegistryService.perspective !== 'login'" class="pad-right-sm">
                 <a matTooltip="Help" href="{{nfRegistryService.documentation.link}}" target="_blank"><i class="fa fa-question-circle help-icon" aria-hidden="true"></i></a>
             </div>
+            <button matTooltip="About" mat-ripple (click)="nfRegistryApi.showRegistryAboutDialog()" mat-icon-button>
```

Comment:
```suggestion
            <button matTooltip="About" mat-ripple (click)="showAboutDialog()" mat-icon-button>
```
Suggestion to restore `showAboutDialog()` method. Pls see my comment about this method in `nf-registry.js`.